pkg/notify/notify_darwin.go

```diff
@@ -6,23 +6,55 @@ import (
 	"context"
 	"os"
 	"os/exec"
+	"strings"
 
 	"github.com/gopasspw/gopass/pkg/ctxutil"
 )
 
+const (
+	terminalNotifier string = "terminal-notifier"
+	osascript        string = "osascript"
+)
+
+var execCommand = exec.Command
+var execLookPath = exec.LookPath
+
 // Notify displays a desktop notification using osascript
 func Notify(ctx context.Context, subj, msg string) error {
 	if os.Getenv("GOPASS_NO_NOTIFY") != "" || !ctxutil.IsNotifications(ctx) {
 		return nil
 	}
-	osas, err := exec.LookPath("osascript")
+
+	//check if terminal-notifier was installed else use the applescript fallback
```

Comment:
please add a space after the comment sign

Reply:
done